NetKAN/WildBlue-PlayMode-USI.netkan

```diff
@@ -0,0 +1,428 @@
+{
+    "spec_version": "v1.18",
+    "identifier":   "WildBlue-PlayMode-USI",
```

Comment:
All other mods have their `name` overridden to `<ModName> <PlayModeName> Play Mode`, except this one. I realize it's not an official WBI mod by Angel125, but I think it would still make sense to follow that scheme for this mod to avoid confusion when looking at the mod list in the client.

Reply:
:heavy_check_mark: